var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -29,6 +29,7 @@ class Llvm(CMakePackage, CudaPackage):
     # fmt: off
     version('master', branch='master')
     version('11.0.0', sha256='8ad4ddbafac4f2c8f2ea523c2c4196f940e8e16f9e635210537582a48622a5d5')
```

Comment:
I think you'll want to explicitly specify `11.0.0` as `preferred=True`, otherwise it might default to `11.0.0-kokkos-tidy`.